cms/djangoapps/contentstore/views/item.py

```diff
@@ -595,6 +595,13 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
             else:
                 duplicate_metadata['display_name'] = _("Duplicate of '{0}'").format(source_item.display_name)
 
+        asides_to_create = []
+        for aside in source_item.runtime.get_asides(source_item):
+            for field in aside.fields.values():
+                if field.scope in (Scope.settings, Scope.content,) and field.is_set_on(aside):
```

Comment:
>This will add the aside to the list once for each field on the aside, right? Will that cause a problem down the road?

In cycle by fields I'm trying to find field that has changes (is different from the default value). After the field is found `aside` is appended to list and `break` instruction points interpreter to take new aside to find changed fields again.

>Also, it will copy fields that aren't Scope.content or Scope.settings, as long as the Aside has at least one Scope.content or Scope.setting field. Is that the intended behavior?

yes. Do I need to change it and copy only `Scope.content` and `Scope.settings` fields?